save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/CloneRepositoryController.kt

```diff
@@ -104,10 +104,13 @@ class CloneRepositoryController(
         } ?: return Mono.just(ResponseEntity.status(HttpStatus.BAD_REQUEST).body("Project doesn't exist"))
     }
 
+    @Suppress("UnsafeCallOnNullableType")
     private fun saveExecution(project: Project, type: ExecutionType, sdk: Sdk): Long {
         val execution = Execution(project, LocalDateTime.now(), null, ExecutionStatus.PENDING, null,
-            null, 0, null, type, null, 0, 0, 0, sdk.toString())
+            null, 0, null, type, null, 0, 0, 0, sdk.toString()).apply {
+            id = executionService.saveExecution(this)
+        }
         log.info("Creating a new execution id=${execution.id} for project id=${project.id}")
-        return executionService.saveExecution(execution)
+        return execution.id!!
```

Comment:
Was there an issue here? I believe, spring-data returns the saved object with id. What's the difference with old version here?

Reply:
Yes, you're right. But in `log.info ()`, before saving, GHHs id is still `null`

Reply:
Oh, ok